src/control/cmd/daos/pool.go

```diff
@@ -79,7 +79,7 @@ func (cmd *poolBaseCmd) connectPool(flags C.uint) error {
 		cmd.log.Debugf("connecting to pool: %s", cmd.PoolID().Label)
 		rc = C.daos_pool_connect2(cLabel, cSysName, flags,
 			&cmd.cPoolHandle, &poolInfo, nil)
-		if rc == 0 {
+		if (rc == 0) {
```

Comment:
```suggestion
		if rc == 0 {
```

Reply:
This is an example of the comment that Jenkins will make on PRs if there are problems.